Docs grammar and typos check (#6064)
vuestorefront/vue-storefront: packages/core/docs/commercetools/composables/use-user-order.md

## Features

> `useUserOrder` composable is responsible for interactions with user order history from your eCommerce backend.

Reviewer: ```suggestion
`useUserOrder` composable is responsible for interactions with customer order history.
```
